rust/src/format/fragment.rs

```diff
@@ -66,7 +66,7 @@ impl From<&pb::DataFile> for DataFile {
 ///
 /// A fragment is a set of files which represent the different columns of the same rows.
 /// If column exists in the schema, but the related file does not exist, treat this column as `nulls`.
-#[derive(Debug, Clone)]
+#[derive(Debug, Clone, PartialEq)]
```

Comment:
Why does it require `PartialEq`, do you need to sort / storing them in some containers?

Reply:
This was for the sake of testing (`assert_eq` requires this).